### ISSUE TITLE
Failig to mock Interfaces with Java 14

### ISSUE THREAD

**Issue by blaszczyk** [jmock-developers/jmock-library#204]
When using Mockery with ByteBuddyClassImportiser (version 2.12.0) to mock any interface with java 14, it fails with an error like:
```
java.lang.IllegalStateException: No code generation strategy available
	at org.jmock.imposters.ByteBuddyClassImposteriser.proxyClass(ByteBuddyClassImposteriser.java:167)
	at org.jmock.imposters.ByteBuddyClassImposteriser.access$000(ByteBuddyClassImposteriser.java:41)
	at org.jmock.imposters.ByteBuddyClassImposteriser$1.apply(ByteBuddyClassImposteriser.java:100)
	at org.jmock.imposters.ByteBuddyClassImposteriser$1.apply(ByteBuddyClassImposteriser.java:96)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1740)
	at org.jmock.imposters.ByteBuddyClassImposteriser.proxy(ByteBuddyClassImposteriser.java:95)
	at org.jmock.imposters.ByteBuddyClassImposteriser.imposterise(ByteBuddyClassImposteriser.java:66)
	at org.jmock.Mockery.mock(Mockery.java:167)
	at org.jmock.Mockery.mock(Mockery.java:143)
```
With java 13 and below mocking classes and interfaces worked well using ByteBuddyClassImportiser. Also Mockery with default Importiser can still mock interfaces in java 14.

**Comment by olibye** [comment]
Thanks, yes for interfaces it would be easier if I used the old mockery
strategy

**Comment by blaszczyk** [comment]
Actually, updating byte buddy manually to the current version (1.10.16) resolved the issue.

**Comment by osparking** [comment]
I added latest bytebuddy dependency into pom.xml and the problem is gone.
(from the default 1.9.? to 
```
		<dependency>
			<groupId>net.bytebuddy</groupId>
			<artifactId>byte-buddy</artifactId>
			<version>1.14.10</version>
		</dependency>
```
)

**Comment by olibye** [comment]
Fixed #252
